verify/src/main/java/com/twilio/verify/domain/TwilioVerifyManager.kt

```diff
@@ -98,4 +98,8 @@ internal class TwilioVerifyManager(
   ) {
     factorFacade.deleteFactor(factorSid, success, error)
   }
+
+  override fun clearLocalData(then: () -> Unit) {
+    factorFacade.clearLocalData(then)
```

Comment:
Do we need to pass this block?

Reply:
Deleting factors (and key pairs for push type) could take a while, my concern is that customers can start using the SDK before all the information was removed

Reply:
But how long? Won't the function exit after all the factors are removed? I believe adding this block adds unneeded complexity

Reply:
Yes, but if it takes a considerable time (depends on number of factors and KeyStore operations), it could block the main thread if it was called from there. Also, for preventing this, we are executing the operation in a background thread, so we are notifying the operation finished